src/openmrs-esm-api.ts

```diff
@@ -1,4 +1,20 @@
+import { defineConfigSchema } from "@openmrs/esm-module-config";
 import "./set-public-path";
+
+defineConfigSchema("@openmrs/esm-api", {
+  redirectAuthFailure: {
+    enabled: {
+      default: true
+    },
+    url: {
+      default: "/openmrs/login.htm"
+    },
+    errors: {
+      default: [401, 403]
```

Comment:
Does the OpenMRS server respond with `403` sometimes when you are not authenticated? I thought it was always a 401, but am not familiar with a lot of the APIs and modules.

Reply:
I have seen 403 in appointments APIs when session times out. But I don't think that should be the common scenario. So removed 403. But still anyone can override in their implementation through config.